src/site/layouts/health_care_local_facility_page.drupal.liquid

```diff
@@ -0,0 +1,178 @@
+    }
+    }
+    }
+    ],
+    "fieldMainLocation": true,
+    "fieldMedia": {
+    "entity": {
+    "image": {
+    "alt": "The entrance to the Pittsburgh VA Medical Center--University Drive",
+    "title": "",
+    "derivative": {
+    "url": "http://dev.va.agile6.com/sites/default/files/styles/crop_3_2/public/2019-02/university-drive-consolidation-building2.jpg?itok=qK823DFr",
+    "width": 480,
+    "height": 330
+    },
+    "entityUrl": {
+    "path": "/pittsburgh-health-care/locations/university-drive-campus"
+    }
+    }
+    }
+    }
+    }
+{% endcomment %}
+
+{% include "src/site/includes/header.html" %}
+{% include "src/site/includes/preview-edit.drupal.liquid" %}
+{% include "src/site/includes/breadcrumbs.drupal.liquid" %}
+<div id="content" class="interior">
+    <main class="va-l-detail-page">
+        <div class="usa-grid usa-grid-full">
+            <nav class="usa-width-one-fourth va-sidebarnav" id="va-detailpage-sidebar">
+                <p>sidebar here</p>
+            </nav>
+            <div class="usa-width-three-fourths">
+                {% if !entityPublished %}
+                    <div class="usa-alert usa-alert-info" >
+                        <div class="usa-alert-body">
+                            <p class="usa-alert-text">You are viewing a draft.</p>
+                        </div>
+                    </div>
+                {% endif %}
+
+                <article class="usa-content vads-l-facility-detail">
+                    {% if title != empty %}
+                        <h1>{{ title }}</h1>
+                    {% endif %}
+
+                    {% if fieldIntroText != empty %}
+                        <div class="va-introtext">
+                            <p>{{ fieldIntroText }}</p>
+                        </div>
+                    {% endif %}
+
+                    <div class="usa-grid usa-grid-full">
+                        <div class="usa-width-one-third">
+                            Make an appointment
+                        </div>
+                        <div class="usa-width-one-third">
+                            View A-Z Health Services
+                        </div>
+                        <div class="usa-width-one-third">
+                            Become a patient
+                        </div>
+                    </div>
+
+
+                    <div class="usa-grid usa-grid-full">
+                        <h2>Our location</h2>
+                        <!-- facility details from api -->
+                        <div class="usa-width-one-half">
+                            <div class="cta-widget" data-widget-type="facility-detail" data-facility="{{ fieldFacilityLocatorApiId | widgetFacilityDetail | escape  }}"></div>
```

Comment:
if we're only sending in the facility id as a string we probably don't need the `escape` filter

Reply:
Same thing here.